src/cloud-api-adaptor/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG BUILD_TYPE=dev
-ARG BUILDER_BASE=quay.io/confidential-containers/golang-fedora:1.21.8-38
+ARG BUILDER_BASE=quay.io/confidential-containers/golang-fedora:1.21.9-38
```

Comment:
Sounds like this is not good:
```
ERROR: failed to solve: quay.io/confidential-containers/golang-fedora:1.21.9-38: failed to resolve source metadata for quay.io/confidential-containers/golang-fedora:1.21.9-38: quay.io/confidential-containers/golang-fedora:1.21.9-38: not found
```

Reply:
OK, let's re-trigger the build if CI fails after merge because https://github.com/confidential-containers/cloud-api-adaptor/blob/main/.github/workflows/build-golang-fedora.yaml might be triggered later than caa.

Reply:
Ok, I'll merge it and then check the tests

Reply:
After I waited for the golang-fedora base image build to happen I triggered the nightly libvirt tests and the podvm image builders passed.